File: library/src/main/java/su/levenetc/android/textsurface/interfaces/ITextEffect.java
package su.levenetc.android.textsurface.interfaces;

import android.graphics.Canvas;
import android.graphics.Paint;

/**
 * Created by Eugene Levenetc.
 */
public interface ITextEffect extends ITextSurfaceAnimation {
	void apply(Canvas canvas, String textValue, float x, float y, Paint paint);
}

File: library/src/main/java/su/levenetc/android/textsurface/interfaces/ITextEffect.kt
package su.levenetc.android.textsurface.interfaces

import android.graphics.Canvas
import android.graphics.Paint

/**
 * Created by Eugene Levenetc.
 */
interface ITextEffect : ITextSurfaceAnimation {
    fun apply(canvas: Canvas, textValue: String, x: Float, y: Float, paint: Paint)
}